namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/pingdom.tf

```diff
@@ -0,0 +1,19 @@
+provider "pingdom" {
+}
+
+resource "pingdom_check" "prepare-a-case-production-check" {
+  type                     = "http"
+  name                     = "${var.application}"
+  host                     = "/https/${var.prepare-case-domain}"
+  resolution               = 1
+  notifywhenbackup         = true
+  sendnotificationwhendown = 6
+  notifyagainevery         = 0
+  url                      = "/"
+  encryption               = true
+  port                     = 443
+  tags                     = "hmpps, prepare-a-case, pic, cloudplatform-managed"
+  probefilters             = "region:EU"
+  integrationids           = []
```

Comment:
I've set up the pingdom/slack integration. The integration ID is 110432, so please add that here when you make the change above.